packages/redux/src/entities/selectors/entity.ts

```diff
@@ -17,17 +19,13 @@ const throwError = (state: StoreState) => {
 /**
  * Gets all entities of a given schema name.
  *
- * @memberof module:entities/selectors
+ * @param state - Application state.
```

Comment:
I believe here we can leave the types as before: `* @param {string} name`. 

What do you think? If fits, apply this to every place that makes sense.

Reply:
I can undo, but it has a warning since this is a TS file.
Since the function is correctly typed the tsDoc correctly types those parameters too.

Reply:
Hello! As the project is now configured with `tsdoc`, we need to remove all `jsdoc` typing information as it is not necessary for `tsdoc`. We have a task to remove them all from the package.

Reply:
Ok, in these scenarios we can remove the jsdoc types :) But in this PR it's not applied the delete of these types in every file we update. Can we have this?

Reply:
I just removed because I was changing the function and I thought "why not". But I can undo it if you prefer to do it in a PR related to that task. Just let me know :)